### PR TITLE
Document configuration parameters for parallel test execution in Jupiter

### PR DESCRIPTION
**Pull request by stoyanK7**
## Overview

Partially addresses #2669. 

Introduces a new table with relevant parallel execution properties in `junit-platform.properties`.

## How the addition looks like

Screenshot from 2022-06-16 19-56-18

---

I hereby agree to the terms of the [JUnit Contributor License Agreement](https://github.com/junit-team/junit5/blob/002a0052926ddee57cf90580fa49bc37e5a72427/CONTRIBUTING.md#junit-contributor-license-agreement).

---

### Definition of Done

- [x] There are no TODOs left in the code
- [x] Method [preconditions](https://junit.org/junit5/docs/snapshot/api/org.junit.platform.commons/org/junit/platform/commons/util/Preconditions.html) are checked and documented in the method's Javadoc
- [x] [Coding conventions](https://github.com/junit-team/junit5/blob/HEAD/CONTRIBUTING.md#coding-conventions) (e.g. for logging) have been followed
- [x] Change is covered by [automated tests](https://github.com/junit-team/junit5/blob/HEAD/CONTRIBUTING.md#tests) including corner cases, errors, and exception handling
- [x] Public API has [Javadoc](https://github.com/junit-team/junit5/blob/HEAD/CONTRIBUTING.md#javadoc) and [`@API` annotations](https://apiguardian-team.github.io/apiguardian/docs/current/api/org/apiguardian/api/API.html)
- [x] Change is documented in the [User Guide](https://junit.org/junit5/docs/snapshot/user-guide/) and [Release Notes](https://junit.org/junit5/docs/snapshot/user-guide/#release-notes)
- [x] All [continuous integration builds](https://github.com/junit-team/junit5#continuous-integration-builds) pass

### CODE REVIEW

File: documentation/src/docs/asciidoc/user-guide/writing-tests.adoc [hunk 1]
```diff
@@ -2333,6 +2333,58 @@ to the same shared resource is running.
 include::{testDir}/example/SharedResourcesDemo.java[tags=user_guide]
 ----
 
+==== Relevant properties
```
Comment (sbrannen) [inline documentation/src/docs/asciidoc/user-guide/writing-tests.adoc:2336]: Please move this content to the `Configuration` section.

File: documentation/src/docs/asciidoc/user-guide/writing-tests.adoc [hunk 2]
```diff
@@ -2333,6 +2333,58 @@ to the same shared resource is running.
...
+
+| ```junit.jupiter.execution.parallel.mode.classes.default```
+|Set execution mode of the top-level classes
+a|
+* `concurrent`
+* `same_thread`
+|```same_thread```
+
+| ```junit.jupiter.execution.parallel.config.strategy```
+|Set execution strategy for desired paralellism and maximum pool size
```
Comment (sbrannen) [inline documentation/src/docs/asciidoc/user-guide/writing-tests.adoc:2365]:
```suggestion
|Set execution strategy for desired parallelism and maximum pool size
```

**Review by sbrannen — CHANGES_REQUESTED** [review]

File: documentation/src/docs/asciidoc/user-guide/writing-tests.adoc (hunk 1 above)
Comment (stoyanK7) [reply documentation/src/docs/asciidoc/user-guide/writing-tests.adoc]: Done!
Screenshot from 2022-06-17 13-07-14

**Review by sbrannen — APPROVED** [review]

**Comment by sbrannen** [comment]
This has been merged into `main` in 3818dcee6f3d5b91b769343ad05e1363abfea6a9 and revised in 2f0a5024e7917c47fac3cfc9d4629ef05470cec8.

Thanks / Blagodaria!

**Pull request closed without merging**